.github/workflows/ci.yml

```diff
@@ -69,10 +69,10 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      # macOS: install gnu-tar because BSD tar is buggy for github actions
+      # macos: install gnu-tar because BSD tar is buggy for github actions
```

Comment:
I guess you didn't fix this? Anyway, thank you for lots of tries.

Reply:
I think it should be fixed now. (I just got the `if` wrong in #1555.)